CHANGELOG.md

```diff
@@ -2,6 +2,63 @@
 
 ## Unreleased
 
+### :new: **New features**
+
+#### Links are easier to read and have a clearer hover state
+
+Links now have underlines that are consistently thinner and a bit further away from the link text.
+
+Links also have a clearer hover state using dark blue (`#003087`) instead of dark pink (`#7c2855`), where the underline gets thicker to make the link stand out to users.
+
+Please carefully review your pages. If you have created your own link styles, you should review them to ensure their use is consistent across your service.
+
+We've also introduced new mixins and modifier classes for:
+
+- text colour links
+- muted colour links
+- links without underlines
+
+This was added in [pull request #1367: Updates to link styles and link hover states](https://github.com/nhsuk/nhsuk-frontend/pull/1367).
+
+#### Style links with text colour
+
+You can now style links with text colour by using either the:
+
+- `nhsuk-link-style-text` mixin in your Sass, or
+- `nhsuk-link--text-colour` class in your HTML
+
+This was added in [pull request #1367: Updates to link styles and link hover states](https://github.com/nhsuk/nhsuk-frontend/pull/1367).
+
+#### Style links with muted colours
+
+You can now style links with muted colours by using either the:
+
+- `nhsuk-link-style-muted` mixin in your Sass, or
+- `nhsuk-link--muted` class in your HTML
+
+This was added in [pull request #1367: Updates to link styles and link hover states](https://github.com/nhsuk/nhsuk-frontend/pull/1367).
+
+#### Style links to remove underlines
+
+You can now remove underlines from links by using either the:
+
+- `nhsuk-link-style-no-underline` mixin in your Sass, or
+- `nhsuk-link--no-underline` class in your HTML
+
```

Comment:
It took me a minute to spot that one of these was a mixin and the other was an HTML class.

Perhaps clearer to separate them a bit more and add `@include` for the mixin?

```suggestion
You can now remove underlines from links by adding the `nhsuk-link--no-underline` HTML class.

If you are creating a custom component you can use the Sass mixin:

```scss
@include nhsuk-link-style-no-underline;
```